src/test/java/hudson/plugins/git/AbstractGitProject.java

```diff
@@ -246,6 +248,11 @@ public String invoke(File f, VirtualChannel channel) throws IOException, Interru
                     throw new RuntimeException(e);
                 }
             }
+
+            @Override
+            public void checkRoles(RoleChecker checker) throws SecurityException {
+                // Nothing to do.
```

Comment:
:bug: no, use `MasterToSlaveFileCallable`. I know this is only in a test, but do not set a bad example for others to copy.